action.yml

```
@@ -9,13 +9,17 @@ inputs:
     description: 'The GitHub token used to create an authenticated client'
     required: true
   hide_user_name:
-    description: 'Github username to hide comments'
+    description: 'GitHub username to hide comments'
     default: 'github-actions[bot]'
     required: false
   hide_reason:
     description: 'Reasons to hide comments'
     default: 'OUTDATED'
     required: false
+  leave_visible:
+    description: 'The number of most recent comments to leave untouched'
+    default: '1'
```

```suggestion
    default: '0'
```

I would like the default to be 0 as I want the same behaviour as existing.

---

That's very acceptable